core/drivers/crypto/caam/include/caam_trace.h

```diff
@@ -35,6 +35,7 @@
 #define DBG_TRACE_HASH	 BIT32(7)  /* Hash trace */
 #define DBG_TRACE_RSA	 BIT32(8)  /* RSA trace */
 #define DBG_TRACE_CIPHER BIT32(9)  /* Cipher dump Buffer */
+#define DBG_TRACE_BLOB   BIT32(10) /* BLOB trace */
```

Comment:
Could you also report this value in comment in ```crypto_conf.mk``` ?
https://github.com/OP-TEE/optee_os/blob/9de9e81542ca568699c81220b563d257d430899b/core/arch/arm/plat-imx/crypto_conf.mk#L11-L23

Reply:
Done in the latest review commit.